src/test/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolderTest.java

```diff
@@ -155,6 +154,30 @@ public void runningBuild() throws Exception {
         d.assertItemNames(5);
     }
 
+    @Test
+    public void notAddChildren() throws Exception {
+        JenkinsRule.WebClient client = r.createWebClient();
+        SampleComputedFolder s = r.jenkins.createProject(SampleComputedFolder.class, "s");
+
+        assertThat(client.getPage(s).getByXPath("//a[contains(text(), \"New Item\")]").size(), is(0));
+
+        s.kids.add("A");
+        s.recompute(Result.SUCCESS);
+
+        assertThat(client.getPage(s).getByXPath("//a[contains(text(), \"New Item\")]").size(), is(0));
+    }
+
+    @Test
+    public void runByTrigger() throws Exception {
+        SampleComputedFolder s = r.jenkins.createProject(SampleComputedFolder.class, "s");
+        s.assertItemNames(0);
+
+        s.addTrigger(new PeriodicFolderTrigger("1m"));
+
+        Thread.sleep(65000);
```

Comment:
I would hope that the periodic calls to `Trigger.run` acc. to `cronTabSpec` are already covered by core tests. `PeriodicFolderTriggerTest` should already be covering `toCrontab` + `toIntervalMillis` + `getInterval`. And since this is not a regression test—unlike JENKINS-33006, there is no known bug you are proving is fixed—it seems to me like a poor tradeoff to make the plugin tests that much slower¹ just for a small bit of added authenticity.

¹Hard to predict by how much—since `concurrency` is on by default, we may just be pausing one “thread” while other test suites run a bit faster on the newly idle CPUs. `ComputedFolderTest` is a suite, run serially, so its individual execution time will be increased by ~70s. If it happens to be the slowest of the last batch of tests to be run, then you will increase `mvn test` time by ~70s, but AFAIK it is random in which order suites are run.